pkgs/games/steam/steamcmd.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchurl, steam, bash
+, steamRoot ? "~/.local/share/Steam"
+}:
+
+stdenv.mkDerivation rec {
+  name = "steamcmd-${version}";
+  version = "20180104"; # According to steamcmd_linux.tar.gz mtime
+
+  src = fetchurl {
+    url = https://steamcdn-a.akamaihd.net/client/installer/steamcmd_linux.tar.gz;
+    sha256 = "0z0y0zqvhydmfc9y9vg5am0vz7m3gbj4l2dwlrfz936hpx301gyf";
+  };
+
+  unpackCmd = ''
```

Comment:
I think the default unpack should work. Any reason for including this?

Reply:
The source tarball does not have a top-level directory. I found a `preUnpack` incantation that works from another package, so hopefully it's a bit cleaner.